### PR TITLE
[daint-gpu] Adding git with subtree check

### PR DESCRIPTION
**Pull request by victorusu**

### CODE REVIEW

**Comment by victorusu** [comment]
@lucamar, nope... I haven't. Should I?

**Comment by lucamar** [comment]
Well, I would try that first, as `git`on Piz Daint is provided by `root`.

File: easybuild/easyconfigs/g/git/git-2.13.1.eb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    ('gettext', '0.19.7'),
+    ('Perl', '5.22.1', '-bare'),
+]
+
+preconfigopts = 'make configure && '
+
+# Work around git build system bug.  If LIBS contains -lpthread, then configure
+# will not append -lpthread to LDFLAGS, but Makefile ignores LIBS.
+configopts = "--with-perl=${EBROOTPERL}/bin/perl --enable-pthreads='-lpthread'"
+#prebuildopts= "  cd contrib/subtree && make install && cd - && "
```
Comment (gppezzi) [inline easybuild/easyconfigs/g/git/git-2.13.1.eb]: can remove this `#prebuildopts` line?

**Comment by gppezzi** [comment]
I suggest we just merge this since the work is already done. 
Next time we'll ask sysadmins for answering such requests related to git.

**Comment by victorusu** [comment]
Can you guys merge this? Thanks.

**Pull request merged**